package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "@lightninglabs/lnc-core",
-  "version": "0.3.1-alpha",
+  "version": "0.3.2-alpha",
   "description": "Type definitions and utilities for Lightning Node Connect",
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "config": {
-    "lnd_release_tag": "v0.17.4-beta",
-    "loop_release_tag": "v0.27.0-beta",
-    "pool_release_tag": "v0.6.4-beta",
-    "faraday_release_tag": "v0.2.11-alpha",
-    "tapd_release_tag": "v0.3.3",
-    "lit_release_tag": "v0.12.3-alpha",
+    "lnd_release_tag": "v0.18.2-beta",
+    "loop_release_tag": "v0.28.5-beta",
+    "pool_release_tag": "4000ec802aaaa9ddcc171f67a4757978fe9101ab",
```

Comment:
@jamaljsr, Let me know if we'd like to get an official tag or branch up for this specific commit instead. I'll request that from the pool team if so.

Reply:
The latest tag in the `pool` repo is `v0.6.5-beta` and I don't seen any proto changes after that tag. So it's safe to generate the TS types using this. I think it would be better to reference a tag instead of a commit because it's much easier for our future selves to understand what's happening, and it won't make a functional difference.

Reply:
The latest tag in the `pool` repo is `v0.6.5-beta` and I don't seen any proto changes after that tag. So it's safe to generate the TS types using this. I think it would be better to reference a tag instead of a commit because it's much easier for our future selves to understand what's happening, and it won't make a functional difference.